exe/how_is

```diff
@@ -23,9 +24,13 @@ elsif options[:version]
   exit
 end
 
+# TODO: Once retroactive issue reports are done, allow exe/how_is to be passed
+#       other dates.
+today = DateTime.now.strftime("%Y-%m-%d")
```

Comment:
Perhaps this is "more clear"?

```ruby
Time.now.to_date.iso8601
```

Reply:
I was going to agree, but then...

```
irb(main):005:0> DateTime.now.iso8601
=> "2017-11-14T03:39:31-05:00"
irb(main):006:0> Time.now.to_date.iso8601
=> "2017-11-14"
irb(main):007:0>
```

now I'm not so sure. I think I'll leave it for now.